docs/_data/sidebar.yml

```diff
@@ -4,9 +4,85 @@ options:
 
     nested_options:
 
-      - title: Index
+      - title: Building Compiler Plugins
         url: /
 
+      - title: Hello World
+        url: /docs/apidocs/compiler-plugin/arrow.meta.plugins.hello-world/hello-world.html
+
+      - title: Hello World IntelliJ IDEA
+        url: /docs/apidocs/idea-plugin/arrow.meta.ide.plugins.nothing/nothing-ide-plugin.html
+
+  - title: Quotes
+
+    nested_options:
+
+      - title: Index
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/index.html
+
+      - title: classOrObject
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/class-or-object.html
+
+      - title: func
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/func.html
+
+      - title: property
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/property.html
+
+      - title: propertyAccessor
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/property-accessor.html
+
+      - title: typeAlias
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/type-alias.html
+
```

Comment:
Nice, are these changes reflected in the kdocs? I was trying to figure out how to start adding pages, but I can just pull this down and work on a branch off yours